OurUmbraco.Site/Views/Master.cshtml

```diff
@@ -90,7 +90,7 @@
     <!--[if lt IE 9]>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/modernizr/2.7.1/modernizr.min.js"></script>
     <![endif]-->
-    <script type="text/javascript" src="https://code.jquery.com/jquery-1.7.2.min.js"></script>
+    <script type="text/javascript" src="https://code.jquery.com/jquery-1.9.1.min.js"></script>
```

Comment:
@nul800sebastiaan is it an issue to update to jQuery version 1.9.1 since Fancybox v3 require this as minimum version https://github.com/fancyapps/fancybox/issues/1400#issuecomment-300514514